src/pages/community-partners.astro

```diff
@@ -0,0 +1,91 @@
+---
+import Layout from '@layouts/Default.astro'
+
+const partners = [
+  {
+    'name': 'TheAlgorithms',
+    'image': 'https://github.com/TheAlgorithms.png',
```

Comment:
```suggestion
    'image': 'https://user-images.githubusercontent.com/51391473/143085576-d0042a60-d718-48d7-ae2e-e2f6edd0db85.gif',
```

Reply:
Hey, I am not changing this, because the GIF looks really weird with the rest of the UI 😄

Reply:
> Hey, I am not changing this, because the GIF looks really weird with the rest of the UI 😄

@vinzvinci, thoughts?

Reply:
I agree with Krish, also if we do that, only The Algorithm has that GIF on the community partners page.